datadog/Example_TorQ_Monitoring_Dashboard.json

```diff
@@ -0,0 +1 @@
parator":">","value":0,"palette":"white_on_green"},{"comparator":"<","value":1,"palette":"white_on_red"}]}],"title":"Data Check - tickerplant1","title_size":"16","title_align":"left","autoscale":true,"precision":0},"layout":{"x":65,"y":9,"width":17,"height":8}},{"id":6,"definition":{"type":"query_value","requests":[{"q":"default_zero(avg:process_wdb1_check{*})","aggregator":"last","conditional_formats":[{"comparator":">=","value":1,"palette":"white_on_green"},{"comparator":"<","value":1,"palette":"white_on_red"}]}],"title":"Data Check - wdb1","title_size":"16","title_align":"left","autoscale":true,"precision":2},"layout":{"x":65,"y":27,"width":17,"height":8}},{"id":7,"definition":{"type":"query_value","requests":[{"q":"default_zero(avg:process_rdb1_check{*})","aggregator":"last","conditional_formats":[{"comparator":">","value":0,"palette":"white_on_green"},{"comparator":"<","value":1,"palette":"white_on_red"}]}],"title":"Data Check - rdb1","title_size":"16","title_align":"left","autoscale":true,"precision":0},"layout":{"x":65,"y":18,"width":17,"height":8}},{"id":8,"definition":{"type":"query_value","requests":[{"q":"avg:system.swap.pct_free{*}","aggregator":"avg","conditional_formats":[{"comparator":">","value":0.15,"palette":"white_on_green"},{"comparator":"<","value":0.15,"palette":"white_on_red"}]}],"title":"Swap Memory Available","title_size":"16","title_align":"left","autoscale":true,"precision":2},"layout":{"x":65,"y":63,"width":17,"height":8}},{"id":9,"definition":{"type":"event_stream","query":"*","event_size":"s","title":"Events Stream","title_size":"16","title_align":"left"},"layout":{"x":83,"y":0,"width":57,"height":23}},{"id":10,"definition":{"type":"query_value","requests":[{"q":"default_zero(avg:process_feed1_check{*})","aggregator":"last","conditional_formats":[{"comparator":">","value":0,"palette":"white_on_green"},{"comparator":"<","value":1,"palette":"white_on_red"}]}],"title":"Data Check - feed1","title_size":"16","title_align":"left","autoscale":true,"precision":2},"layout":{"x":65,"y":0,"width":17,"height":8}},{"id":11,"definition":{"type":"query_value","requests":[{"q":"default_zero(avg:process_hdb2_check{*})","aggregator":"last","conditional_formats":[{"comparator":">","value":0,"palette":"white_on_green"},{"comparator":"<","value":1,"palette":"white_on_red"}]}],"title":"Data Check - hdb2","title_size":"16","title_align":"left","autoscale":true,"precision":2},"layout":{"x":65,"y":45,"width":17,"height":8}},{"id":12,"definition":{"type":"query_value","requests":[{"q":"default_zero(avg:process_gateway1_check{*})","aggregator":"last","conditional_formats":[{"comparator":">","value":0,"palette":"white_on_green"},{"comparator":"<","value":1,"palette":"white_on_red"}]}],"title":"Data Check - gateway1","title_size":"16","title_align":"left","autoscale":true,"precision":2},"layout":{"x":65,"y":54,"width":17,"height":8}}],"template_variables":[],"layout_type":"free","is_read_only":false,"notify_list":[],"dashboard_id":"era-ztf-vye"}
```

Comment:
is this file meant to be read/edited manually? if so, can prettify the layout with some JSON formatter

Reply:
The contents are just pasted in, or the file is uploaded to the website,  and then any changes to the dash board can be done on the website.